app/views/content/blog/application-tips-from-a-teacher-training-provider.md

```diff
@@ -0,0 +1,66 @@
+
+## The personal statement
+
+The personal statement is your shop window. You will be asked to focus on why you have decided to train to teach and why you are suited to your chosen age group or subject. You can also talk about any relevant work experience you have had, what you learned or observed and how you might apply this to your own teaching practice.
+
+Remember to tailor your statement to either the phase or the subject you are wanting to teach. Trying to adapt it to cover both primary and secondary, or two different subjects, is not advised.
+
+Your personal statement is also an opportunity to acknowledge any gaps in your subject knowledge that you think you might need to plug. Not every provider will automatically consider a [Subject Knowledge Enhancement (SKE)](improve-your-subject-knowledge) course when reading an application, particularly if it’s for an application for something that is outside of your degree discipline. Take this opportunity to suggest that you would be happy to undertake a SKE course to enhance your existing knowledge ahead of starting your teacher training course. This shows a commitment to your own professional development.
+
+You may wish to demonstrate your awareness of the hard work and commitment required to undertake teaching as a career but also demonstrate how excited you are to start your journey.
+
+I can’t emphasise enough how important it is to avoid spelling mistakes. Make sure to run it through a spell check before copying it into your application, as demonstrating good communication skills is vital. Read more [tips on your personal statement.](/tips-on-applying-for-teacher-training#your-personal-statement)
+
+## References
+
+And finally, we turn to references. Who you pick is really important. You will need two references. One could be a university or academic tutor and the second could be from a school or somewhere you have worked in a voluntary capacity, or it could be someone from your current employment.
+
+Whoever you choose, you should contact your references before you start writing your application so that they are aware that they will be contacted and that a swift response is important. If your references are not ready this can hold up your application. Read more about [choosing your referees.](/tips-on-applying-for-teacher-training#choose-your-referees)
+
+## Responses
+
+The good news is that as soon as you press the Submit button on your application then it will appear in your chosen providers’ inboxes. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.
```

Comment:
```suggestion
The good news is that as soon as you press the Submit button on your application then it will appear in your chosen providers’ inbox. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.
```

Reply:
Also 'Submit' being capitalised doesn't feel quite right maybe
```suggestion
The good news is that as soon as you submit your application then it will appear in your chosen providers’ inboxes. Please do not expect an instant response, particularly if you are applying for School Direct places, as the lead school who handles the application process on behalf of a partnership may have to contact your chosen school and wait for them to consider your application and set up an interview. This can take some time to organise, particularly when some schools have two-week timetables.
```